gen/definitions/generic/service_lan_vpn_ethernet_interface_feature_associate_tracker.yaml

```diff
@@ -0,0 +1,290 @@
+---
+name: Service LAN VPN Interface Ethernet Feature Associate Tracker Feature
+rest_endpoint: /v1/feature-profile/sdwan/service/%v/lan/vpn/%s/interface/ethernet/%s/tracker/
+minimum_version: 20.12.0
+id_attribute: parcelId
+doc_category: Features - Service
+test_tags: [SDWAN_2012]
+attributes:
+  - tf_name: feature_profile_id
+    reference: true
+    type: String
+    mandatory: true
+    description: Feature Profile ID
+    example: f6dd22c8-0b4f-496c-9a0b-6813d1f8b8ac
+    test_value: sdwan_service_feature_profile.test.id
+  - tf_name: service_lan_vpn_feature_id
+    reference: true
+    type: String
+    mandatory: true
+    description: Service LAN VPN Feature ID
+    example: 140331f6-5418-4755-a059-13c77eb96037
+    test_value: sdwan_service_lan_vpn_feature.test.id
+  - tf_name: service_lan_vpn_interface_ethernet_feature_id
+    reference: true
+    type: String
+    mandatory: true
+    description: Service LAN VPN Interface Ethernet Feature ID
+    example: 140331f6-5418-4755-a059-13c77eb96037
+    test_value: sdwan_service_lan_vpn_interface_ethernet_feature.test.id
+  - model_name: parcelId
+    tf_name: service_tracker_feature_id
+    type: String
+    mandatory: true
+    description: Service Tracker Feature ID
+    example: 140331f6-5418-4755-a059-13c77eb96037
+    test_value: sdwan_service_tracker_feature.test.id
+
+test_prerequisites: |
+  resource "sdwan_service_feature_profile" "test" {
+    name        = "TF_TEST"
+    description = "Terraform test"
+  }
+
+  resource "sdwan_service_lan_vpn_feature" "test" {
```

Comment:
Can we work with minimal resource definitions for test pre-requisites, as I don't think we need to configure static routes for example to make this test work? In other words, remove as many of the resource attributes as possible.